modules/log/event.go

```diff
@@ -359,11 +359,15 @@ func (m *MultiChannelledLog) Flush() {
 
 // GetLevel gets the level of this MultiChannelledLog
 func (m *MultiChannelledLog) GetLevel() Level {
+	m.mutex.Lock()
```

Comment:
imho `mutex` should be `RWMutex` and Get methods use `RLock` and `RUnlock`

Reply:
The RWmutex requires two calls to lock and unlock and would be slower.

This is simply a lock around a single value. If performance were really an issue an `atomic.Load*` `atomic.Store*` might be quicker but this.

Reply:
What do you me two calls? It requires same number of calls but state read only calls is not locking each other